xbmc/utils/CMakeLists.txt

```diff
@@ -187,9 +188,29 @@ if(CORE_SYSTEM_NAME STREQUAL android OR AML_FOUND)
   list(APPEND HEADERS AMLUtils.h)
 endif()
 
-if(AML_FOUND)
-  list(APPEND HEADERS ScreenshotAML.cpp)
-  list(APPEND HEADERS ScreenshotAML.h)
+if(CORE_PLATFORM_NAME_LC STREQUAL aml)
+  list(APPEND HEADERS ScreenshotSurfaceAML.cpp)
+  list(APPEND HEADERS ScreenshotSurfaceAML.h)
+endif()
+
+if(CORE_PLATFORM_NAME_LC STREQUAL rbpi)
+  list(APPEND HEADERS ScreenshotSurfaceRBP.cpp)
+  list(APPEND HEADERS ScreenshotSurfaceRBP.h)
+endif()
+
+if(CORE_PLATFORM_NAME_LC STREQUAL windows)
+  list(APPEND HEADERS ScreenshotSurfaceWin32.cpp)
+  list(APPEND HEADERS ScreenshotSurfaceWin32.h)
```

Comment:
rename to `ScreenshotSurfaceWindows` or something similar as this file is for desktop (win32) and uwp